clang/lib/3C/CastPlacement.cpp

```diff
@@ -46,7 +46,7 @@ bool CastPlacementVisitor::VisitCallExpr(CallExpr *CE) {
               TypeVars[TyVar->GetIndex()] != nullptr)
             ArgExpr = ArgExpr->IgnoreImpCasts();
 
-          CVarSet ArgumentConstraints = CR.getExprConstraintVars(ArgExpr);
+          CVarSet ArgumentConstraints = CR.getExprConstraintVarsSet(ArgExpr);
```

Comment:
Seems likely to conflict with stuff @jackastner is doing in https://github.com/correctcomputation/checkedc-clang/pull/356